docker.Makefile

```diff
@@ -128,22 +128,15 @@ build-e2e-image:
 	IMAGE_NAME=$(E2E_IMAGE_NAME) VERSION=$(VERSION) docker buildx bake e2e-image
 
 .PHONY: test-e2e
-test-e2e: test-e2e-non-experimental test-e2e-experimental test-e2e-connhelper-ssh ## run all e2e tests
+test-e2e: test-e2e-always-experimental test-e2e-connhelper-ssh ## run all e2e tests
 
-.PHONY: test-e2e-experimental
-test-e2e-experimental: build-e2e-image # run experimental e2e tests
+.PHONY: test-e2e-always-experimental
+test-e2e-always-experimental: build-e2e-image # run non-experimental e2e tests
 	docker run --rm $(ENVVARS) -e DOCKERD_EXPERIMENTAL=1 \
```

Comment:
Having a quick look, and it looks like the `scripts/test/e2e/run` that you updated was the only location where this env-var was used, so we can remove it here;

```suggestion
	docker run --rm $(ENVVARS) \
```